extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/sql_load_script/InvalidMultilineSqlLoadScriptTestCase.java

```diff
@@ -5,22 +5,20 @@
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.quarkus.hibernate.orm.MyEntity;
 import io.quarkus.test.QuarkusUnitTest;
 
 public class InvalidMultilineSqlLoadScriptTestCase {
-    @RegisterExtension
+    //@RegisterExtension
     static QuarkusUnitTest runner = new QuarkusUnitTest()
             .setExpectedException(PersistenceException.class)
             .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
                     .addAsResource("application-invalid-multiline-test.properties", "application.properties")
                     .addAsResource("invalid-multiline.sql")
                     .addClasses(MyEntity.class));
 
-    @Test
+    //@Test
```

Comment:
I think https://github.com/quarkusio/quarkus/pull/4252 should fix this so you could rebase and revert this part.

Reply:
ah I had a similar comment and did not see that you requested the change a few minutes earlier.